src/igvfd/schemas/donor.json

```diff
@@ -36,7 +36,7 @@
     ],
     "properties": {
         "schema_version": {
-            "default": "1"
+            "default": "3"
```

Comment:
Minor question here. I think in the other abstract parent schemas, we don't put a default schema version (such as in https://github.com/IGVF-DACC/igvfd/blob/dev/src/igvfd/schemas/sample.json#L43). Should we do that here too?

Reply:
I changed this to a hidden comment to match sample.json.